### ISSUE TITLE
Required input attr

### ISSUE THREAD

**Issue by CezarLuiz0** [ubilabs/react-geosuggest#80]
When I put `required` nothing happens

```javascript
<Geosuggest
  placeholder="Start location"
  ref="start_location"
  required
/>
```

**Comment by ro-ka** [comment]
I suppose there should be a required attribute attached to the input, right?

**Comment by CezarLuiz0** [comment]
Yes, the result code would be:

```
<div class="geosuggest ">
    <input class="geosuggest__input " type="text" value="" placeholder="Start location" required>
    <ul class="geosuggest__suggests geosuggest__suggests--hidden"></ul></div>
```

You can use spread operator, something like `<input ... {...this.props}`

**Comment by ro-ka** [comment]
Ah, okay, will add.

**Comment by CezarLuiz0** [comment]
Something like:

```javascript
<input
          className={'geosuggest__input ' + this.props.inputClassName}
          ref="geosuggestInput"
          type="text"
          value={this.state.userInput}
          placeholder={this.props.placeholder}
          disabled={this.props.disabled}
          onKeyDown={this.onInputKeyDown}
          onChange={this.onInputChange}
          onFocus={this.onFocus}
          onBlur={this.hideSuggests}
          {...this.props}
 />
```

**Comment by ro-ka** [comment]
I wouldn’t put all the props on the element. I think a whitelist with supported attributes it the better way.

**Comment by CezarLuiz0** [comment]
You can create other prop like `isRequired={true}`, for example.

```
<Geosuggest isRequired />
```

**Comment by wedneyyuri** [comment]
@ro-ka Are you already working on this? I also need this feature.

**Comment by wedneyyuri** [comment]
@ro-ka What do you think about including other attributes like `autocomplete`, `name`, `id`, etc...

The solution proposed by @CezarLuiz0 seems to solve this problem, in my opinion the component should not know all the properties assigned to the input element.

**Comment by ro-ka** [comment]
Will add, yes. No time till now. :)
That’s true, but it’ll attach a lot of non standard attributes to the input element. Will a whitelist with attributes that are defined in the spec to add.

**Comment by ro-ka** [comment]
Released as `1.15.0`. Thanks for bringing this up!

**Comment by CezarLuiz0** [comment]
Awesome! :+1:

**Comment by wedneyyuri** [comment]
@ro-ka I think it's missing the `id` attribute.

That's the case:
```
<label for="place">Search Places</label>
```

**Comment by ro-ka** [comment]
Totally, will fix that!

**Comment by ro-ka** [comment]
Fixed in `1.15.1`. Thanks! :)
